packages/core/admin/admin/src/content-manager/components/BlocksEditor/BlocksInput/index.js

```diff
@@ -74,6 +74,35 @@ const BlocksInput = ({ disabled, placeholder }) => {
     }
   };
 
+  /**
+   * Modifier keyboard shortcuts
+   */
+  const handleKeyboardShortcuts = (event) => {
+    const isMac = /Mac|iPod|iPhone|iPad/.test(navigator.userAgent);
+    const isCtrlOrCmd = isMac ? event.metaKey : event.ctrlKey;
+
+    if (isCtrlOrCmd) {
+      switch (event.key) {
+        case 'b':
+          modifiers.bold.handleToggle();
+          break;
+        case 'i':
+          modifiers.italic.handleToggle();
+          break;
+        case 'u':
+          modifiers.underline.handleToggle();
+          break;
+        case 'e':
+          modifiers.code.handleToggle();
+          break;
+        case 'S':
+          if (event.shiftKey) modifiers.strikethrough.handleToggle();
```

Comment:
I'm curious how you picked this shortcut?

Reply:
Yannis suggested it, https://strapi-inc.atlassian.net/browse/CS-254?focusedCommentId=17288